src/components/AttachmentModal.js

```diff
@@ -264,15 +277,23 @@ class AttachmentModal extends PureComponent {
                         ) : ''}
                     />
                     <View style={styles.imageModalImageCenterContainer}>
-                        {this.state.sourceURL && (
-                            <AttachmentView
-                                sourceURL={sourceURL}
-                                file={this.state.file}
-                                onToggleKeyboard={this.updateConfirmButtonVisibility}
+                        {this.state.reportId ? (
+                            <AttachmentCarousel
+                                showArrows={this.state.showArrows}
+                                reportId={this.state.reportId}
+                                onNavigate={this.onNavigate}
+                                sourceURL={this.props.sourceURL}
```

Comment:
is there a reason we use the props.sourceURL here instead of state.sourceURL as used in AttachmentView? It would be nice to just use one if possible.

Reply:
it's used to initialize the data in the carousel by matching the report's url. I added to the comment on the sourceURL prop